monitoring/uptime-robot/readme.md

```diff
@@ -0,0 +1,59 @@
+# Uptime Robot Monitoring
+
+Monitoring status page can be found here: https://status.developer.gov.bc.ca/
+
+This is a high level monitoring aiming to provide the community with a sense of our service reliability. Monitored services include the DevOps OpenShift 4 Platforms and the shared services. Uptime Robot tracks a history of service uptime and incidence list for each of them. In addition, there will be announcements regarding each major service downtime and maintenance window.
+
+Please also note that in addition to Uptime Robot monitoring, there are more sophisticated metrics and checks that the team is using for each individual shared services and OCP clusters.
+
+## Monitors and Alerts
+
+| Monitor | Endpoint | Monitoring Interval | Alerts |
+| ------- |--------- | ------------------- | ------ |
+| Gold / Silver Cluster | readyz & Cerberus** | 1 min | RC / MSTeams / SMS / Email |
+| Klab / Clab / ARO Cluster | readyz & Cerberus** | 1 min | RC |
+| SSO (prod/test/dev) | service URL | 1 min | RC |
+| RocketChat | service URL | 1 min | RC / MSTeams |
+| Artifactory | service API ping | 1 min | RC / MSTeams |
+| Aqua | TBD | TBD | TBD |
+| Vault | TBD | TBD | TBD |
+
+***Please note*** that 1 minute interval (shortest available from Uptime Robot) sometime is considered a bit frequent for busy services. But for statistic purpose, we'd like to keep is as short as possible to not mistakenly record each brief downtime including the long wait time util next monitoring round. To address this problem, with a 1 minute interval, we will only receive alerts when service is down for 5 minutes. This is okay as Platform Services Team is not replying solely on Uptime Robot for monitoring. At the time when team gets notified from Uptime Robot, there would be other monitoring mechanism triggered already, such as Sysdig dashboards and alerts.
+
+** Cerberus: is a RedHat suggested monitoring tool for OCP cluster general healthiness. For more details, see doc [here](../cerberus/readme.md).
+
+## Alert Integrations
+
+When a service is down for more than 5 minutes, the preset alerts will be fire off. Here are some major types of alert we are currently using:
+
+**1. RocketChat and MSTeams:**
+- webhook setup in a notification channel
+- service lead is tagged from the message
+- MSTeams used as a backup strategy when RC is affected by cluster wide issue
+
+**2. SMS:**
+- text message send to service lead for immediate response
+
+**3. Email:**
+- for cluster downtime alerts
+- ***Note:*** team should create a custom Uptime Robot announcement after cluster issue being resolved. See next section for details.
+
+
+## Announcements
+
+There are different types of announcements you will see from Uptime Robot:
+
+**1. Automatic Announcement:**
+- whenever there is a downtime detected by Uptime Robot, it will auto generate an announcement with the timestamp and duration
```

Comment:
```suggestion
- whenever there is a service downtime detected by Uptime Robot, it will auto generate an announcement with the timestamp and duration
```